apps/riscv-tests/isa/rv64uv/vsuxei.c

```diff
@@ -0,0 +1,141 @@
+// Copyright 2021 ETH Zurich and University of Bologna.
+// Solderpad Hardware License, Version 0.51, see LICENSE for details.
+// SPDX-License-Identifier: SHL-0.51
+//
+// Author: Matteo Perotti <mperotti@iis.ee.ethz.ch>
+
+#include "vector_macros.h"
+
+#define AXI_DWIDTH 128
+
+#define INIT 98
+
+void reset_vec8(volatile uint8_t *vec, int rst_val, uint64_t len) {
+  for (uint64_t i = 0; i < len; ++i)
+    vec[i] = rst_val;
+}
+void reset_vec16(volatile uint16_t *vec, int rst_val, uint64_t len) {
+  for (uint64_t i = 0; i < len; ++i)
+    vec[i] = rst_val;
+}
+void reset_vec32(volatile uint32_t *vec, int rst_val, uint64_t len) {
+  for (uint64_t i = 0; i < len; ++i)
+    vec[i] = rst_val;
+}
+void reset_vec64(volatile uint64_t *vec, int rst_val, uint64_t len) {
+  for (uint64_t i = 0; i < len; ++i)
+    vec[i] = rst_val;
+}
```

Comment:
Do you use those functions?

Reply:
Yes, to reset the vectors to a known state value, to check if the masked stores work.